### PR TITLE
Edits to native routing section

### PR DESCRIPTION
**Pull request by bfallonf**
@mrunalp this is in response to #470 . Can I ask for a quick tech review? Not much has changed. A few questions though:
1. Is the "Network Overlay" section describing the diagram? Or maybe is it describing the end result of the "Node Setup" procedure? Also, the 11.11.0.0/16 address is mentioned a few times, but it's not on the diagram... Should it be?
2. In the "Router Setup" intro para, what is meant by "they"? IE: "They would have to be modified to use the syntax for the particular router." Does they mean the Linux box? The NICs? The routers? 

I think that's all. Thanks!

### CODE REVIEW

**Comment by mrunalp** [comment]
@bfallonf LGTM

>> Is the "Network Overlay" section describing the diagram? Or maybe is it describing the end result of the "Node Setup" procedure? Also, the 11.11.0.0/16 address is mentioned a few times, but it's not on the diagram... Should it be?

The description is more generic than the diagram. I don't think we need to add 11.11.0.0/16 to the diagram. 

>> In the "Router Setup" intro para, what is meant by "they"? IE: "They would have to be modified to use the syntax for the particular router." Does they mean the Linux box? The NICs? The routers? 

"They" refers to steps and commands to be executed on the router in use.

**Comment by bfallonf** [comment]
@mrunalp Ok. Thanks for the extra info. I've added/edited some parts to reflect that. I'll put this over to peer review.

File: admin_guide/native_container_routing.adoc [hunk 1]
```diff
@@ -10,58 +10,77 @@
...
+$ brctl addbr lbr0
+$ ip addr add 11.11.1.1/24 dev lbr0
+$ ip link set dev lbr0 up
 ----
-* Modify docker startup script to use the new bridge (/etc/sysconfig/docker on Fedora).
+
+. Modify the docker startup script to use the new bridge. By default, the
+startup script is found in the `/etc/sysconfig/docker` directory:
++
 ----
```
Comment (tnguyen-rh) [inline admin_guide/native_container_routing.adoc]: On my system `/etc/sysconfig/docker` is a file.  I believe it is the script referenced in the first sentence.

**Comment by tnguyen-rh** [comment]
Last nit: Are the example commands executable by non-root user?  If not, then the prompts should be changed to `#`.

**Comment by bfallonf** [comment]
@tnguyen-rh thanks for the edits. Suggestions made into edits.

If nothing else, this should be ok to merge.

**Comment by adellape** [comment]
LGTM.

**Pull request merged**